ddtrace/bootstrap/sitecustomize.py

```diff
@@ -79,9 +55,6 @@ def update_patched_modules():
 
     opts = {}
 
-    if enabled and enabled.lower() == "false":
-        opts["enabled"] = False
-        patch = False
```

Comment:
Note that this changes functionality: in `ddtrace-run` if `DATADOG_TRACE_ENABLED` was set to false then no patching would be done.

Now the behaviour is consistent with what is done if the tracer is disabled in general, which is to still generate spans but not send them to the agent.

Reply:
hmm, this could cause problems for people.... ?

Reply:
In theory, yes... If we're fine keeping the asymmetry between ddtrace-run and the base library we can leave it as-is which might be the safer option right now. I really don't like that the same variable has different meanings though.

Reply:
I agree with you, but I'd prefer if we submit as a separate PR. separation of concerns/changes. would be clearer in release notes, and be easier to roll back as a single change.

Reply:
:+1: agreed